### DISCUSSION TITLE
is drawing over yAxis possible ?

### DISCUSSION THREAD

**Discussion by MoeQatoum** [pyqtgraph/pyqtgraph#2128] [Q&A] [answered]
hi, 
I'm trying to draw a textLabel over the yAxis (like the crosshair lines but on the yAxis), I understand that I should overlay a view over the yAxis I tried `PlotItem.layout.addItem` but that fires a warning `QGridLayoutEngine::addItem: Cell (2, 2) already taken` and affect axis width for some reason, I looked every where read the documentation , couldn't manage to find a solution. I would appreciate a small example ... thanks

**Comment by danielhrisca** [comment, marked as answer]
[here](https://github.com/danielhrisca/asammdf/blob/1507f100bddc86c5f037db2770ec39e42bdfc306/asammdf/gui/widgets/formated_axis.py#L35) I add two buttons on top of the y axis. I think you could add a text in a similar way
